lib/SILOptimizer/Analysis/ARCAnalysis.cpp

```diff
@@ -964,7 +963,7 @@ void ConsumedArgToEpilogueReleaseMatcher::collectMatchingReleases(
     if (!isa<DestroyAddrInst>(inst)) {
       // We do not know if we have a fully post dominating release
       // set, so we use the partial post dom entry point.
-      if (auto partialReleases = iter->second.getPartiallyPostDomReleases()) {
+      if (auto partialReleases = ArgInstMap[arg].getPartiallyPostDomReleases()) {
```

Comment:
We don't want to make this change. We are in this case doing two different look ups. We want to do a single lookup.

Reply:
Thanks, I reverted changes to places where # of look ups are significant.